teku/src/test/java/tech/pegasys/teku/cli/subcommand/RemoteSpecLoaderTest.java

```diff
@@ -55,6 +64,29 @@ void shouldFailWhenRequiredItemsAreMissing() {
         .hasMessageContaining("GENESIS_FORK_VERSION");
   }
 
+  @Test
+  void shouldProvideValidSpecConfigWithIncompleteRemoteConfig() throws IOException {
+    final String jsonConfig =
+        Resources.toString(
+            Resources.getResource(RemoteSpecLoaderTest.class, "spec_config.json"),
+            StandardCharsets.UTF_8);
+    final ObjectMapper objectMapper = new ObjectMapper();
+    TypeReference<Map<String, String>> typeReference = new TypeReference<>() {};
+    Map<String, String> data = objectMapper.readValue(jsonConfig, typeReference);
+
+    final SpecConfig specConfig = SpecConfigLoader.loadRemoteConfig(data);
+
+    final SpecConfig expectedConfig =
```

Comment:
basically instead of doing this, you'd just create a mainnet spec, and get the genesis spec config from that...